### PR TITLE
Update regex to support composite file names

### PR DESCRIPTION
**Pull request by Ethan-Arrowood**
Fixes bug introduced in #37

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ const console = require('console')
 
 const DISPLAY_NAME_SYMBOL = Symbol.for('fastify.display-name')
 const fpStackTracePattern = new RegExp('at\\s{1}plugin\\s{1}.*\\n\\s*(.*)')
-const fileNamePattern = new RegExp('\\/(\\w*)\\.js')
+const fileNamePattern = new RegExp('\\/(\\w*)(\\.\\w*)?\\.js')
```
Comment (mcollina) [inline index.js]: I would prefer if we include hits in the previous group, basically "anything that is not a /". Note that the file name could also include numbers.

Can you add a unit test?

Comment (Ethan-Arrowood) [reply index.js]: If a file is named: `myPlugin123.whatever.js`
Should the appropriate `'fastify.display-name'` be: `myPlugin123.whatever` or `myPlugin123`?

Comment (mcollina) [reply index.js]: > A unit test would require specially named files. Is that okay?

Yes.

> If a file is named: myPlugin123.whatever.js
Should the appropriate 'fastify.display-name' be: myPlugin123.whatever or myPlugin123?

it should be `myplugin123.whatever`.

Comment (Ethan-Arrowood) [reply index.js]: Great! Will do.

**Comment by mcollina** [comment]
Can you move all tests into a `test` folder?

**Pull request merged**